plugins/view-transitions/includes/settings.php

```diff
@@ -126,6 +130,7 @@ function plvt_sanitize_setting( $input ): array {
 		'post_title_selector',
 		'post_thumbnail_selector',
 		'post_content_selector',
+		'admin_transition_animation',
```

Comment:
Better sanitize this to a boolean outside of this list. For example the other fields here use `sanitize_text_field()` which doesn't really make sense for a checkbox.